test/acceptance/authz/autoschema_test.go

```diff
@@ -0,0 +1,185 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/weaviate/weaviate/client/authz"
+	"github.com/weaviate/weaviate/client/objects"
+	"github.com/weaviate/weaviate/entities/models"
+	"github.com/weaviate/weaviate/entities/schema"
+	"github.com/weaviate/weaviate/test/helper"
+	"github.com/weaviate/weaviate/usecases/auth/authorization"
+)
+
+func TestAutoschemaAuthZ(t *testing.T) {
+	customUser := "custom-user"
+	customKey := "custom-key"
+	adminKey := "admin-key"
+	adminUser := "admin-user"
+	adminAuth := helper.CreateAuth(adminKey)
+
+	readSchemaAction := authorization.ReadCollections
+	createDataAction := authorization.CreateData
+	updateSchemaAction := authorization.UpdateCollections
+	createSchemaAction := authorization.CreateCollections
+
+	_, down := composeUp(t, map[string]string{adminUser: adminKey}, map[string]string{customUser: customKey}, nil)
```

Comment:
This function doesn't append the `WithAutoSchema` builder that you've introduced so is this test actually running with auto schema? Maybe auto schema is on by default if not set?